src/compare/computeMeanDifference.ts

```diff
@@ -0,0 +1,34 @@
+import { Image } from '..';
+
+/**
+ * Compute the mean difference between two images.
+ * Ignore values that are zero in both images.
```

Comment:
Do you have a reference for this behavior?

Reply:
As this is not a commonly used algorithm, it is removed from `image-js`. The algorithm was copied to this issue: https://github.com/cheminfo/profid-padif/issues/20